tests/Unit/Evolution/Systems/NewtonianEuler/Test_Characteristics.cpp

```diff
@@ -26,10 +26,13 @@ namespace {
 
 template <size_t Dim>
 void test_characteristic_speeds(const DataVector& used_for_size) noexcept {
```

Comment:
Should this be marked noexcept? I think we've been leaving out noexcept for things that call pypp.

Reply:
We've been inconsistent in tests whether or not we mark things `noexcept`. We agreed long ago that it's not required in testing code but also not wrong to do so. Another good question!